From 070e0bda6405d44e2ff51120c8906623657d2a5c Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Thu, 12 Apr 2018 17:30:16 +0200
Subject: [PATCH 70/84] base-files: remove /etc/uci-defaults/11_migrate-sysctl

11_migrate-sysctl has not been updated with new file hashes since 2012.
Let's get rid of it.

Signed-off-by: Matthias Schiffer <mschiffer@universe-factory.net>
---
 .../files/etc/uci-defaults/11_migrate-sysctl     | 16 ----------------
 1 file changed, 16 deletions(-)
 delete mode 100644 package/base-files/files/etc/uci-defaults/11_migrate-sysctl

diff --git a/package/base-files/files/etc/uci-defaults/11_migrate-sysctl b/package/base-files/files/etc/uci-defaults/11_migrate-sysctl
deleted file mode 100644
index 464e275779..0000000000
--- a/package/base-files/files/etc/uci-defaults/11_migrate-sysctl
+++ /dev/null
@@ -1,16 +0,0 @@
-#!/bin/sh
-
-if [ ! -f "/rom/etc/sysctl.conf" ] || cmp -s "/rom/etc/sysctl.conf" "/etc/sysctl.conf"; then
-	exit 0
-fi
-
-fingerprint="$(md5sum /etc/sysctl.conf)"
-fingerprint="${fingerprint%% *}"
-
-if [ "$fingerprint" = "1b05ebb41f72cb84e5510573cd4aca26" ] || \
-   [ "$fingerprint" = "62deb895be1a7f496040187b7c930e4e" ]; then
-	logger -t migrate-sysctl "Updating sysctl.conf to use current defaults"
-	cp "/rom/etc/sysctl.conf" "/etc/sysctl.conf"
-fi
-
-exit 0
-- 
2.17.1

